client/src/index.js

```diff
@@ -7,4 +7,4 @@ createRoot(document.getElementById("root")).render(
 	<BrowserRouter>
 		<App />
 	</BrowserRouter>
-);
+);
```

Comment:
Remove this change from the commit

Reply:
please can you remind me again the command I am supposed to undo this

Reply:
thank you